CyberArk/cyberark-audit-logs/tests/PAM00099.json

```diff
@@ -0,0 +1,43 @@
+{
+  "input": {
+    "message": "{\"uuid\": \"fe2b3e00-d8f9-4942-aa63-5fcaebc489f2\", \"tenantId\": \"43de6333-65f1-4626-aeec-2cff238e61ca\", \"timestamp\": 1739266337571, \"username\": \"john.doe@cyberark.cloud.xxxxx\", \"applicationCode\": \"PAM\", \"auditCode\": \"PAM00099\", \"auditType\": \"Info\", \"action\": \"Open File\", \"userId\": \"john.doe@cyberark.cloud.xxxxx\", \"source\": \"PVWA\", \"actionType\": \"Execute\", \"component\": \"Vault\", \"serviceName\": \"Privilege Cloud\", \"accessMethod\": null, \"accountId\": null, \"target\": \"\", \"command\": null, \"sessionId\": null, \"message\": \"\", \"customData\": {\"PAM\": {\"new_target\": \"\", \"target\": \"Root\\\\PVConfiguration.xml\"}}, \"cloudProvider\": null, \"cloudWorkspacesAndRoles\": [], \"cloudIdentities\": null, \"cloudAssets\": null, \"safe\": \"PVWAConfig\", \"accountName\": \"\", \"targetPlatform\": \"\", \"targetAccount\": \"\", \"identityType\": null}"
+  },
+  "expected": {
+    "message": "{\"uuid\": \"fe2b3e00-d8f9-4942-aa63-5fcaebc489f2\", \"tenantId\": \"43de6333-65f1-4626-aeec-2cff238e61ca\", \"timestamp\": 1739266337571, \"username\": \"john.doe@cyberark.cloud.xxxxx\", \"applicationCode\": \"PAM\", \"auditCode\": \"PAM00099\", \"auditType\": \"Info\", \"action\": \"Open File\", \"userId\": \"john.doe@cyberark.cloud.xxxxx\", \"source\": \"PVWA\", \"actionType\": \"Execute\", \"component\": \"Vault\", \"serviceName\": \"Privilege Cloud\", \"accessMethod\": null, \"accountId\": null, \"target\": \"\", \"command\": null, \"sessionId\": null, \"message\": \"\", \"customData\": {\"PAM\": {\"new_target\": \"\", \"target\": \"Root\\\\PVConfiguration.xml\"}}, \"cloudProvider\": null, \"cloudWorkspacesAndRoles\": [], \"cloudIdentities\": null, \"cloudAssets\": null, \"safe\": \"PVWAConfig\", \"accountName\": \"\", \"targetPlatform\": \"\", \"targetAccount\": \"\", \"identityType\": null}",
+    "event": {
+      "action": "Open File",
+      "category": [
+        "configuration"
+      ],
+      "code": "PAM00099",
+      "dataset": "PAM",
+      "provider": "Vault",
+      "type": [
+        "access"
+      ]
+    },
+    "@timestamp": "2025-02-11T09:32:17.571000Z",
+    "cyberark": {
+      "audit": {
+        "safe": "PVWAConfig",
+        "uuid": "fe2b3e00-d8f9-4942-aa63-5fcaebc489f2"
+      }
+    },
+    "file": {
+      "name": "PVConfiguration.xml",
+      "path": "Root"
```

Comment:
My bad. `Root\\PVConfiguration.xml` is a better value. Can you fix the parser? Thank you